docker/datahub-frontend/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM openjdk:8 as builder
 
 RUN apt-get update && apt-get install -y wget \
     && wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb \
-    && dpkg -i google-chrome-stable_current_amd64.deb; apt-get -fy install
+    && dpkg -i google-chrome-stable_current_amd64.deb; apt-get -fy install; dpkg -i google-chrome-stable_current_amd64.deb
```

Comment:
This seems a little bit like a hack. I wonder if it's the order of commands that leads to the race condition. Can you try swap `dpkg` & `apt-get` instead of running `dpkg` twice to see if it fixes the issue?

Reply:
Fixed, no it's even less hacky than the original